ports/stm32/boards/ARDUINO_GIGA/pins.csv

```diff
@@ -1,3 +1,110 @@
+D0,PB7
+D1,PA9
+D2,PA3
+D3,PA2
+D4,PJ8
+D5,PA7
+D6,PD13
+D7,PB4
+D8,PB8
+D9,PB9
+D10,PK1
+D11,PJ10
+D12,PJ11
+D13,PH6
+D14,PG14
+D15,PC7
+D16,PH13
+D17,PI9
+D18,PD5
+D19,PD6
+D20,PB11
+D21,PH4
+D22,PJ12
+D23,PG13
+D24,PG12
+D25,PJ0
+D26,PJ14
+D27,PJ1
+D28,PJ15
+D29,PJ2
+D30,PK3
+D31,PJ3
+D32,PK4
+D33,PJ4
+D34,PK5
+D35,PJ5
+D36,PK6
+D37,PJ6
+D38,PJ7
+D39,PI14
+D40,PE6
+D41,PK7
+D42,PI15
+D43,PI10
+D44,PG10
+D45,PI13
+D46,PH15
+D47,PB2
+D48,PK0
+D49,PE4
+D50,PI11
+D51,PE5
+D52,PK2
+D53,PG7
+D54,PI5
+D55,PH8
+D56,PA6
+D57,PJ9
+D58,PI7
+D59,PI6
+D60,PI4
+D61,PH14
+D62,PG11
+D63,PH11
+D64,PH10
+D65,PH9
+D66,PA1
+D67,PD4
+D68,PC6
+D69,PI0
+D70,PI1
+D71,PI2
+D72,PI3
+D73,PC1
+D74,PB12
+D75,PD3
+D76,PC4
+D77,PC5
+D78,PB0
+D79,PB1
+D80,PC3
+D81,PC2
+D82,PC0
+D83,PA0
+D84,PA4
+D85,PA5
+D86,PI12
+D87,PJ13
+D88,PE3
```

Comment:
Added additional pins, based on the variant.cpp file for the [GIGA variant](https://github.com/arduino/ArduinoCore-mbed/blob/68fbece79947ce2a64bf49f7ba5f5f8111d89454/variants/GIGA/variant.cpp#L133-L140). These are present on the SPI header (J7).

```suggestion
D88,PE3
D89,PG9
D90,PD7
D91,PB3
D92,PA15
```